specs/Standards/Tokens/FungibleTokenCore.md

```diff
@@ -0,0 +1,356 @@
+   - `dex` calls `wbtc::transfer({"new_owner_id": "alice", "amount": "8000000000"})`
+   - `dex` calls `wltc::transfer({"new_owner_id": "bob", "amount": "900100000000"})`
+
+## Reference-level explanation
+
+**NOTES**:
+- All amounts, balances and allowance are limited by `U128` (max value `2**128 - 1`).
+- Token standard uses JSON for serialization of arguments and results.
+- Amounts in arguments and results have are serialized as Base-10 strings, e.g. `"100"`. This is done to avoid JSON limitation of max integer value of `2**53`.
+- The contract must track the change in storage when adding to and removing from collections. This is not included in this core fungible token standard but instead in the [Storage Standard](../Storage.md).
+- To prevent the deployed contract from being modified or deleted, it should not have any access keys on its account.
+
+**Interface**:
+
+```javascript
+/************************************/
+/* CHANGE METHODS on fungible token */
+/************************************/
+// Simple transfer to a receiver. Does not call a method.
+// Requirements:
+// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes
+// * Caller must have greater than or equal to the `amount` being requested
+// `receiver_id` is the valid NEAR account receiving the fungible tokens.
+// `amount` is the number of tokens to transfer, wrapped in quotes and treated like a string, although the number will be stored as an unsigned integer with 128 bits.
+// The `memo` argument is optional. It's added for use cases that may benefit from indexing or providing information for a transfer.
+function ft_transfer(
+    receiver_id: string,
+    amount: string,
+    memo: string|null
+): void {}
+
+// Transfer tokens and call a method on a receiver contract. A successful workflow will end in a success execution outcome to the callback on the same contract at the method `ft_resolve_transfer`.
+//
+// You can think of this as being similar to attaching native NEAR tokens to a function call on some contract. It allows you to attach any Fungible Token in a call to a receiver contract, given by `receiver_id`, where `msg` gets passed to the receiver contract to indicate both a function to call and the parameters to that function.
+// Requirements:
+// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes
+// * Caller must have greater than or equal to the `amount` being requested
+// * The receiving contract must implement `ft_on_transfer` according to the standard.
+// * This contract must implement `ft_resolve_transfer` according to the standard.
+// `msg` is an argument that may specify any information expected by the receiving contract in order to properly handle the function. If the receiving contract needs something larger than a single string, `msg` could be a complex JSON structure, base64-encoded.
```

Comment:
```suggestion
// `msg` is an argument that may specify any information expected by the receiving contract in order to properly handle the function. You can think about it as input arguments for the receiving contract to process this transfer. If the receiving contract needs something larger than a single string, `msg` could be a complex JSON structure, or a base64-encoded binary data.
```

Reply:
Handled an alternative way in https://github.com/near/NEPs/commit/34b68f1aa576657e8f50381b20c26c2f467a5852